composer.json

```diff
@@ -16,7 +16,7 @@
     "homepage": "https://www.pdfparser.org",
     "require": {
         "php": ">=5.6",
-        "ext-mbstring": "*",
```

Comment:
Why removing this requirement? IMHO this should be kept to outline that this library depends on the extension (even with a polyfill). Or doesn't it make sense when using PHP 8?

Reply:
I think as we added the polyfill, the mbstring extension is no longer a requirement, strictly speaking. Sure, it will be used if it's there, but the environment is not required to have it installed or activated in order to be able to run pdfparser.

I don't know if there are any environments doing this, but I could imagine a scenario where a script could check the composer.json for (extension) requirements and refuse the installation of pdfparser (or a depending package) when running on PHP 8 or missing the mbstring extension for some other reason.